src/api/index.yaml

```diff
@@ -387,6 +387,13 @@ paths:
             - lines
             - points
           default: lines
+        - name: wrapLongitudes
+          in: query
+          description: |
```

Comment:
I couldn't understand what this parameter does without going over the code. I would change the description to something like `When false, deviate from the geojson standard and don't wrap longitudes between [-180, 180], so that tracks going over the antimeridean line don't reset back to the negated longitudes. This should be false if you are using an atlantic-centered visualization, such as QGis, or when using leaflet or mapbox to render the track. See https://macwright.org/2016/09/26/the-180th-meridian.html.`

In addition to this, shouldn't this be false by default, so that we don't wrap by default? I think most use cases will use the unwrapped format (@nerik says QGis supports unwrapped longitudes, as do mapbox and leaflet).

Reply:
Thanks for the description, your is much clearer and I'll try to put more attention next time.

I set it true by default to keep the same behaviour we used to have and not introduce a breaking change in the API because we weren't sure it works everywhere, for example the R package some data people is working with. could you @enriquetuya confirm this?

If we can do that, I'd be glad of set as default behaviour too

Reply:
@j8seangel Nobody else is using the API (yet) besides the data portal, the public map and the carrier portal, and sometimes myself when I load a track through QGIS. I think we are safe to change the default behavior here. I'm a bit concerned over people forgetting this parameter exists and then complaining that the tracks are all broken in a couple of months when they start using this from the R package.

Reply:
sounds good, I wasn't full aware of who is using this API and we also prefer having this as a default behaviour